static/styles/portico-signin.css

```diff
@@ -105,7 +105,23 @@ html {
     transform: none;
     text-align: center;
 }
+.back-to-login {
```

Comment:
There should be new lines before and after each CSS selector.

Also, I'm not so sure that you should be creating a new class selector with the same exact properties as a different selector? I would rather just create a new selector to go in `components.css` or something so that classes can be reused between both Back to login and Forgot password? links.